### PR TITLE
docs: fix docs of vpc_v1

### PR DESCRIPTION
**Pull request by Zhukun-Huawei**
docs: fix docs of vpc_v1

### CODE REVIEW

File: docs/data-sources/vpc_route_ids_v2.md
Comment (chengxiangdong) [inline docs/data-sources/vpc_route_ids_v2.md:1]: The field types in this document are missing, please complete them.

File: docs/data-sources/vpc_route_v2.md
Comment (chengxiangdong) [inline docs/data-sources/vpc_route_v2.md:1]: 1. The field types in this document are missing, please complete them.
2. The argument `region` is usually in the first position.

File: docs/data-sources/vpc_subnet_ids_v1.md
Comment (chengxiangdong) [inline docs/data-sources/vpc_subnet_ids_v1.md:1]: 1. The field types in this document are missing, please complete them.
2. The argument region is usually in the first position.

File: docs/data-sources/vpc_subnet_v1.md
Comment (chengxiangdong) [inline docs/data-sources/vpc_subnet_v1.md:1]: 1. The field types in this document are missing, please complete them.
2. The argument region is usually in the first position.

File: docs/data-sources/vpcep_endpoints.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ data "flexibleengine_vpcep_endpoints" "dns_endpoints" {
 ## Attributes Reference
 
 In addition to all arguments above, the following attributes are exported:
-
+``
```
Comment (chengxiangdong) [inline docs/data-sources/vpcep_endpoints.md:32]: Please remove redundant content

File: docs/data-sources/vpcep_endpoints.md [hunk 2]
```diff
@@ -50,6 +50,8 @@ The `endpoints` block contains:
 * `whitelist` - List of IP or CIDR block which can access the VPC endpoint.
 * `private_domain_name` - DNS name pointing to the VPC endpoint ip.
 * `tags` - The key/value pairs to associate with the VPC endpoint.
+* `tags.key` - Specifies the tag key. Each tag key contains a maximum of 127 unicode characters but cannot be left blank.
+* `tags.value` - Specifies the tag value list. Each value contains a maximum of 255 Unicode characters. Before using values, delete SBC spaces before and after the value.
```
Comment (chengxiangdong) [inline docs/data-sources/vpcep_endpoints.md:54]:
```suggestion
* `tags` - The key/value pairs to associate with the VPC endpoint.
  + `key` - The tag key. Each tag key contains a maximum of 127 unicode characters but cannot be left blank.
  + `value` - The tag value list. Each value contains a maximum of 255 Unicode characters. Before using values, delete SBC spaces before and after the value.
```

File: docs/data-sources/vpcep_public_services.md [hunk 3]
```diff
@@ -24,14 +24,14 @@ data "flexibleengine_vpcep_public_services" "dns_service" {
 
 * `service_id` - (Optional, String) Specifies the unique ID of the public VPC endpoint service.
 
+* `region` - The region in which to obtain the public VPC endpoint services.
```
Comment (chengxiangdong) [inline docs/data-sources/vpcep_public_services.md:27]:
```suggestion
* `region` - (Optional, String) The region in which to obtain the public VPC endpoint services.
  If omitted, the provider-level region will be used.
```

File: docs/resources/networking_secgroup_v2.md
Comment (chengxiangdong) [inline docs/resources/networking_secgroup_v2.md:1]: In this doc, the field type and `ForceNew` are missing, please complete them.

File: docs/resources/vpc_peering_connection_accepter_v2.md [hunk 4]
```diff
@@ -77,6 +77,9 @@ The following arguments are supported:
 
 * `accept` (Optional)- Whether or not to accept the peering request. Defaults to `false`.
 
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the vpc subnet. If omitted, the
+  provider-level region will be used. Changing this creates a new subnet.
```
Comment (chengxiangdong) [inline docs/resources/vpc_peering_connection_accepter_v2.md:81]:
```suggestion
* `region` - (Optional, String, ForceNew) Specifies the region in which to create the VPC peering connection accepter.
  If omitted, the provider-level region will be used. Changing this creates a new VPC perring connection.
```

File: docs/resources/vpc_peering_connection_accepter_v2.md
Comment (chengxiangdong) [inline docs/resources/vpc_peering_connection_accepter_v2.md:1]: In this doc, the field type and `ForceNew` are missing, please complete them.

File: docs/resources/vpc_peering_connection_accepter_v2.md (hunk 4 above)
Comment (chengxiangdong) [reply docs/resources/vpc_peering_connection_accepter_v2.md:81]: Please move it to the first position in the argument

File: docs/resources/vpc_peering_connection_v2.md [hunk 5]
```diff
@@ -43,6 +43,9 @@ The following arguments are supported:
 * `peer_vpc_id` (Required) - Specifies the VPC ID of the accepter tenant. Changing this creates a new VPC peering connection.
 
 * `peer_tenant_id` (Optional) - Specified the Tenant Id of the accepter tenant. Changing this creates a new VPC peering connection.
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the vpc subnet. If omitted, the
+  provider-level region will be used. Changing this creates a new subnet.
```
Comment (chengxiangdong) [inline docs/resources/vpc_peering_connection_v2.md:48]:
```suggestion
* `region` - (Optional, String, ForceNew) Specifies the region in which to create the VPC peering connection.
  If omitted, the provider-level region will be used. Changing this creates a new subnet.
```

Please move it to the first position in the argument

File: docs/resources/vpc_peering_connection_v2.md
Comment (chengxiangdong) [inline docs/resources/vpc_peering_connection_v2.md:1]: In this doc, the field type and `ForceNew` are missing, please complete them.

File: docs/resources/vpc_route_v2.md
Comment (chengxiangdong) [inline docs/resources/vpc_route_v2.md:1]: In this doc, the field type and `ForceNew` are missing, please complete them.

File: docs/resources/vpc_route_v2.md [hunk 6]
```diff
@@ -25,6 +25,9 @@ resource "flexibleengine_vpc_route_v2" "vpc_route" {
 
 The following arguments are supported:
 
+* `region` - (Optional, String, ForceNew) The region in which to create the vpc route table.
+  If omitted, the provider-level region will be used. Changing this creates a new resource.
```
Comment (chengxiangdong) [inline docs/resources/vpc_route_v2.md:29]:
```suggestion
* `region` - (Optional, String, ForceNew) The region in which to create the VPC route table.
  If omitted, the provider-level region will be used. Changing this creates a new resource.
```

File: docs/resources/vpcep_approval.md [hunk 7]
```diff
@@ -66,14 +66,14 @@ The following arguments are supported:
 * `endpoints` (Required) - Specifies the list of VPC endpoint IDs which accepted to connect to VPC endpoint service.
     The VPC endpoints will be rejected when the resource was destroyed.
 
+* `region` - The region in which to obtain the VPC endpoint service.
+
```
Comment (chengxiangdong) [inline docs/resources/vpcep_approval.md:70]:
```suggestion
* `region` - (Optional, String, ForceNew) The region in which to create the resource.
  If omitted, the provider-level region will be used. Changing this creates a new resource.
```

Please move it to the first position in the parameter

File: docs/resources/vpcep_endpoint.md [hunk 8]
```diff
@@ -104,14 +104,14 @@ The following arguments are supported:
 
 * `tags` - (Optional) The key/value pairs to associate with the VPC endpoint.
 
+* `region` - The region in which to create the VPC endpoint.
```
Comment (chengxiangdong) [inline docs/resources/vpcep_endpoint.md:107]:
```suggestion
* `region` - (Optional, String, ForceNew) The region in which to create the resource.
  If omitted, the provider-level region will be used. Changing this creates a new resource.
```

Please move it to the first position in the parameter

File: docs/resources/vpcep_endpoint.md
Comment (chengxiangdong) [inline docs/resources/vpcep_endpoint.md:1]: In this docs, the field type and `ForceNew` properties are missing, please fill them in.

File: docs/resources/vpcep_approval.md
Comment (chengxiangdong) [inline docs/resources/vpcep_approval.md:1]: In this docs, the field type and `ForceNew` properties are missing, please fill them in.

File: docs/resources/vpcep_service.md
Comment (chengxiangdong) [inline docs/resources/vpcep_service.md:1]: 1. In this docs, the field type and `ForceNew` properties are missing, please fill them in.
2. The argument `region` is usually in the first position.

**Review by chengxiangdong — CHANGES_REQUESTED** [review]

File: docs/data-sources/networking_port.md [hunk 9]
```diff
@@ -17,6 +17,14 @@ data "flexibleengine_networking_port" "port_1" {
...
 ## Argument Reference
 
+* `admin_state_up` - The administrative state of the port.
+
+* `device_id` - The ID of the device the port belongs to.
+
+* `device_owner` - The device owner of the port.
+
+* `name` - The name of the port.
+
```
[the remark this answers is not in the document]
Comment (Zhukun-Huawei) [reply docs/data-sources/networking_port.md:30]: > Please move them after the region field.

**Review by Zhukun-Huawei — COMMENTED** [review]
done

File: docs/data-sources/networking_port.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (Zhukun-Huawei) [reply docs/data-sources/networking_port.md]: > please move the attribute into `Attributes Reference`

image
done

File: docs/data-sources/vpc_peering_connection_v2.md
[the remark this answers is not in the document]
Comment (Zhukun-Huawei) [reply docs/data-sources/vpc_peering_connection_v2.md:1]: done

File: docs/resources/networking_secgroup_rule_v2.md
[the remark this answers is not in the document]
Comment (Zhukun-Huawei) [reply docs/resources/networking_secgroup_rule_v2.md:1]: done

File: docs/data-sources/vpc_route_ids_v2.md [hunk 10]
```diff
@@ -33,6 +33,9 @@ output "route_nexthop" {
 
 * `vpc_id` (Required) - The VPC ID that you want to filter from.
 
+* `region` - (Optional) The region in which to obtain the V1 VPC client. A VPC client is needed to retrieve VPCs.
+  If omitted, the region argument of the provider is used.
+
```
[the remark this answers is not in the document]
Comment (Zhukun-Huawei) [reply docs/data-sources/vpc_route_ids_v2.md]: done

File: docs/data-sources/vpc_route_v2.md
Comment (Zhukun-Huawei) [reply docs/data-sources/vpc_route_v2.md:1]: done

File: docs/data-sources/vpc_subnet_ids_v1.md
Comment (Zhukun-Huawei) [reply docs/data-sources/vpc_subnet_ids_v1.md:1]: done

File: docs/data-sources/vpc_subnet_v1.md
Comment (Zhukun-Huawei) [reply docs/data-sources/vpc_subnet_v1.md:1]: done

File: docs/data-sources/vpcep_endpoints.md (hunk 1 above)
Comment (Zhukun-Huawei) [reply docs/data-sources/vpcep_endpoints.md]: done

File: docs/data-sources/vpcep_endpoints.md (hunk 2 above)
Comment (Zhukun-Huawei) [reply docs/data-sources/vpcep_endpoints.md]: done

File: docs/data-sources/vpcep_public_services.md (hunk 3 above)
Comment (Zhukun-Huawei) [reply docs/data-sources/vpcep_public_services.md]: done

File: docs/resources/networking_secgroup_v2.md
Comment (Zhukun-Huawei) [reply docs/resources/networking_secgroup_v2.md:1]: done

File: docs/resources/vpc_route_v2.md (hunk 6 above)
Comment (Zhukun-Huawei) [reply docs/resources/vpc_route_v2.md]: done

File: docs/resources/vpcep_approval.md (hunk 7 above)
Comment (Zhukun-Huawei) [reply docs/resources/vpcep_approval.md]: done

File: docs/resources/vpcep_service.md [hunk 11]
```diff
@@ -35,62 +35,70 @@ resource "flexibleengine_vpcep_service" "demo" {
...
-* `service_port` - (Optional) Specifies the port for accessing the VPC endpoint service.
+* `service_port` - (Optional, Int) Specifies the port for accessing the VPC endpoint service.
     This port is provided by the backend service to provide services. The value ranges from 1 to 65535.
 
-* `terminal_port` - (Optional) Specifies the port for accessing the VPC endpoint.
+* `terminal_port` - (Optional, Int) Specifies the port for accessing the VPC endpoint.
     This port is provided by the VPC endpoint, allowing you to access the VPC endpoint service.
     The value ranges from 1 to 65535.
 
+* `service_type` - (Required, String, ForceNew)The type of the VPC endpoint service. Only **interface** can be configured.
```
Comment (ShiChangkuo) [inline docs/resources/vpcep_service.md:77]: `service_type` is optional in schema, please check it carefully.

Comment (Zhukun-Huawei) [reply docs/resources/vpcep_service.md]: revised

File: docs/resources/vpcep_endpoint.md [hunk 12]
```diff
@@ -81,37 +81,38 @@ resource "flexibleengine_vpcep_endpoint" "demo" {
...
-* `ip_address` (Optional) - Specifies the IP address for accessing the associated VPC endpoint service.
+* `ip_address` - (Optional, String, ForceNew) Specifies the IP address for accessing the associated VPC endpoint service.
     Only IPv4 addresses are supported. Changing this creates a new VPC endpoint.
 
-* `enable_dns` (Optional) - Specifies whether to create a private domain name. The default value is true.
+* `enable_dns` - (Optional, Bool, ForceNew) Specifies whether to create a private domain name. The default value is true.
     Changing this creates a new VPC endpoint.
 
-* `enable_whitelist` (Optional) - Specifies whether to enable access control. The default value is false.
+* `enable_whitelist` (Optional, Bool) - Specifies whether to enable access control. The default value is false.
```
Comment (chengxiangdong) [inline docs/resources/vpcep_endpoint.md:102]:
```suggestion
* `enable_whitelist` (Optional, Bool, ForceNew) - Specifies whether to enable access control. The default value is false.
```

File: docs/resources/vpcep_endpoint.md [hunk 13]
```diff
@@ -81,37 +81,38 @@ resource "flexibleengine_vpcep_endpoint" "demo" {
...
     Changing this creates a new VPC endpoint.
 
-* `whitelist` (Optional) - Specifies the list of IP address or CIDR block which can be accessed to the VPC endpoint.
+* `whitelist` (Optional, List) - Specifies the list of IP address or CIDR block which can be accessed to the VPC endpoint.
```
Comment (chengxiangdong) [inline docs/resources/vpcep_endpoint.md:105]:
```suggestion
* `whitelist` (Optional, List, ForceNew) - Specifies the list of IP address or CIDR block which can be accessed to the VPC endpoint.
```

File: docs/resources/vpc_flow_log_v1.md
[the remark this answers is not in the document]
Comment (chengxiangdong) [reply docs/resources/vpc_flow_log_v1.md:1]: pls remove `status`

File: docs/data-sources/vpc_peering_connection_v2.md
Comment (chengxiangdong) [inline docs/data-sources/vpc_peering_connection_v2.md:1]: pls remove attribute `description`

File: docs/data-sources/vpc_route_v2.md
Comment (chengxiangdong) [inline docs/data-sources/vpc_route_v2.md:1]: pls move `nexthop` to argument.

File: docs/data-sources/vpc_subnet_v1.md [hunk 14]
```diff
@@ -57,6 +60,8 @@ the selected subnet.
 
 * `dhcp_enable` - DHCP function for the subnet.
 
+* `description` - The description of the subnet.
```
Comment (chengxiangdong) [inline docs/data-sources/vpc_subnet_v1.md:63]: pls move `description` to argument

**Pull request merged**